finvizfinance/screener/overview.py

```diff
@@ -15,7 +15,7 @@
     NUMBER_COL,
     util_dict,
 )
-
+from time import sleep
```

Comment:
Please move the import up to follow code style requirement.

Reply:
Yes, i have moved up before this finviz finance import, but we need sleep. Because it will block us on continuous calls.